docker/armorblox/.gitignore

```diff
@@ -0,0 +1 @@
+requirements.txt
```

Comment:
If we are copying requirements.txt in the Dockerfile, how can we ignore the file?

Reply:
This is how all the images directory look like. They support `pipfiles` and the requirement.txt is added from `pipenv` only So, may be they don not want to git maintain it